src/BSD-1-Clause.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+  <license licenseId="BSD-1-Clause" name="BSD 1-clause License">
+    <crossRefs>
+      <crossRef>https://svnweb.freebsd.org/base/head/include/ifaddrs.h?revision=326823</crossRef>
+    </crossRefs>
+    <copyrightText>
+      <p>
+        Copyright (c)<alt name="copyright" match=".+"><year> <owner></alt>
+        All rights reserved.
+      </p>
+    </copyrightText>
+    <p>
+      Redistribution and use in source and binary
+      forms, with or without modification, are permitted
+      provided that the following conditions are met:
+    </p>
+    <list>
+      <item>
+        <bullet>1.</bullet>
+        Redistributions of source code must retain the above copyright
+        notice, this list of conditions and the following disclaimer.
+      </item>
+    </list>
+    <p>
+      THIS SOFTWARE IS PROVIDED BY<alt name="copyrightHolderAsIs"
+      match=".+">Berkeley Software Design, Inc.</alt>
```

Comment:
I think it's odd to template `<owner>` in the Copyright line, but not template it here.  And as long as we're templating it, it would be nice to use `THE COPYRIGHT HOLDERS AND CONTRIBUTORS` (like BSD-2-Clause) or some other generic text as the default here.